NEWS.md

```diff
@@ -1,3 +1,7 @@
+# CHANGES IN GGIR VERSION 2.9-8
```

Comment:
These updates will be part of a future 2.9-7 release. I merged PR #861 into the master after making the 2.9-6 release. I did that because 2.9-6 had already enough code changes and I considered PR #861 to be in need for a few more weeks of testing, which is why it is in the master branch but not in the release yet.